src/highlevel_api.jl

```diff
@@ -295,3 +295,15 @@ function _bytes2string(x::Integer)
 end
 
 has_object_of_type(t) = any(obj -> obj.type_ == t, gettopology())
+
+"""
+Shows a graphical visualization of the system topology.
+The quality of the result might depend on the used terminal and might vary between machines and operating systems.
+
+**Note:** The specific visualization may change between minor versions.
+"""
+function topology_graphical()
+    lstopo_ng = joinpath(dirname(Hwloc_jll.libhwloc), "..", "bin", "lstopo-no-graphics")
```

Comment:
Uhm, we should rather list explicitly as products of the JLL the binaries you need

Reply:
Yeah, I was lazy here :) Once https://github.com/JuliaPackaging/Yggdrasil/pull/5579 is merged I'll use the following instead:
```julia
Hwloc_jll.lstopo_no_graphics() do lstopo
    run(`$lstopo --no-io --no-legend --of txt`)
end;
```